files/default/sql/ddl/updates/1.0.0.sql

```diff
@@ -13,4 +13,287 @@ CREATE TABLE IF NOT EXISTS `secrets` (
+                                               `hopsworks`.`cached_feature_group`(`id`)
+                                               ON DELETE CASCADE
+                                               ON UPDATE NO ACTION;
+
+ALTER TABLE `hopsworks`.`training_dataset` ADD  COLUMN `training_dataset_type`   INT(11) NOT NULL DEFAULT '0';
+
+ALTER TABLE `hopsworks`.`training_dataset` ADD COLUMN `hopsfs_training_dataset_id` INT(11) NULL;
+ALTER TABLE `hopsworks`.`training_dataset` ADD CONSTRAINT `hopsfs_training_dataset_fk`
+                                                FOREIGN KEY (`hopsfs_training_dataset_id`) REFERENCES
+                                               `hopsworks`.`hopsfs_training_dataset`(`id`)
+                                               ON DELETE CASCADE
+                                               ON UPDATE NO ACTION;
+
+ALTER TABLE `hopsworks`.`training_dataset` ADD COLUMN `external_training_dataset_id` INT(11) NULL;
+ALTER TABLE `hopsworks`.`training_dataset` ADD CONSTRAINT `external_training_dataset_fk`
+                                                FOREIGN KEY (`external_training_dataset_id`) REFERENCES
+                                               `hopsworks`.`external_training_dataset`(`id`)
+                                               ON DELETE CASCADE
+                                               ON UPDATE NO ACTION;
+
+
+/*
+  Move columns from feature_group to cached_feature_group
+*/
+
+-- Move hive_tbl_id
+INSERT INTO `hopsworks`.`cached_feature_group`(`offline_feature_group`)
+SELECT `hive_tbl_id` FROM `hopsworks`.`feature_group` WHERE feature_group_type=0;
+
+-- Set foreign key on parent table
+UPDATE `hopsworks`.`feature_group` INNER JOIN `hopsworks`.`cached_feature_group`
+    ON `feature_group`.`hive_tbl_id` = `cached_feature_group`.`offline_feature_group`
+SET `feature_group`.`cached_feature_group_id` = `cached_feature_group`.`id`;
+
+/*
+  Drop foreign keys and columns from the parent table that was move to the child table
+*/
+
+-- drop foreign key to metastore.TBLS
+SET @fk_name = (SELECT CONSTRAINT_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE TABLE_SCHEMA = "hopsworks" AND TABLE_NAME = "feature_group" AND REFERENCED_TABLE_NAME="TBLS");
+SET @s := concat('ALTER TABLE hopsworks.feature_group DROP FOREIGN KEY `', @fk_name, '`');
+PREPARE stmt1 FROM @s;
+EXECUTE stmt1;
+DEALLOCATE PREPARE stmt1;
+
+-- Drop hive_tbl_id column from feature_group
+ALTER TABLE `hopsworks`.`feature_group` DROP COLUMN `hive_tbl_id`;
+
+/*
+  Move columns from training_dataset to hopsfs_training_dataset
+*/
+
+-- Move Inode
+INSERT INTO `hopsworks`.`hopsfs_training_dataset`(`inode_pid`, `inode_name`, `partition_id`)
+SELECT `inode_pid`, `inode_name`, `partition_id` FROM `hopsworks`.`training_dataset` WHERE training_dataset_type=0;
+
+-- Set foreign key on parent table
+UPDATE `hopsworks`.`training_dataset` INNER JOIN `hopsworks`.`hopsfs_training_dataset`
```

Comment:
Same as above. I don't think you are allowed to have UPDATE statements without where condition

Reply:
See question above

Reply:
Done